package.json

```diff
@@ -8,9 +8,16 @@
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.24.0",
     "react": "^17.0.2",
+    "react-browser-router": "^2.1.2",
+    "react-burger-menu": "^3.0.6",
     "react-dom": "^17.0.2",
+    "react-router": "^6.0.0",
+    "react-router-dom": "^6.0.0",
     "react-scripts": "4.0.3",
-    "router": "^1.3.5"
+    "react-slick": "^0.28.1",
+    "react-slideshow-image": "^3.6.0",
+    "router": "^1.3.5",
+    "slick-carousel": "^1.8.1"
```

Comment:
For several of these packags, I want you to check you use them (some look like errors to me, but I may be wrong)